resources/webcourses/src/app/pages/webcourse/activities/sidebar/shared/bookmark-card/bookmark-card.component.html

```diff
@@ -0,0 +1,4 @@
+<a role="button" class="row border justify-content-between bg-light" (click)="this.router.navigateByUrl('/webcourse/activities/'+bookmark.aid)">
```

Comment:
- [ ] Needs centered vertical alignment
- [ ] Replace activity ID with garbage can icon, to allow deleting bookmark without having to go to the activity
- [ ] Add second line under bookmark description which includes in date created and activity ID, format e.g. "`13 April, 2013 at 13:13 (Activity #13)`... class should be "text-uppercase text-secondary" in a reduced font size

Reply:
Okay why are we using these code snippets, some of these requests are for a different component